hr-time/idlharness.html

```diff
@@ -15,24 +15,41 @@
 <div id="log"></div>
 
 <pre id='untested_idl' style='display:none'>
-
 interface Window {
 };
 
-interface Performance {
+[Constructor(DOMString scriptURL), Exposed=(Window,Worker)]
```

Comment:
Seems the testing framework doesn't handle `Exposed=(Window,Worker)` properly.

Existing tests in this repository either commented it out, or split it into windows and worker tests (#1989).